examples/shared_samples/logical_interconnect.rb

```diff
@@ -56,7 +58,8 @@
   name:  'li_et01',
   smartLink:  false,
   privateNetwork:  false,
-  connectionTemplateUri: nil
+  connectionTemplateUri: nil,
+  type: 'ethernet-networkV4'
```

Comment:
we don't need type here as we are setting default values in lib.

Reply:
I'll change this. Forgot to delete after running the example.